### ISSUE TITLE
User has no profile

### ISSUE THREAD

**Issue by peterdesmet** [inbo/vespa-watch#52]
Since I pulled and migrated I get an error logging in:

```
RelatedObjectDoesNotExist at /admin/login/
User has no profile.
```

I think it is related to https://github.com/inbo/vespa-watch/issues/43, where my user account has profile?

**Comment by bartaelterman** [comment]
Log in to your admin screen, edit your user and set a zone for your user.
This shall be done automatically when we create user accounts for production. But the user in your database was created before the profile existed.

**Comment by peterdesmet** [comment]
Thanks for the reply

> Log in to your admin screen, edit your user and set a zone for your user.

Cannot do this for my peter_desmet user as that generates the error I described.

**Comment by bartaelterman** [comment]
As Nico pointed out, there is probably indeed a bug that needs to be fixed and setting a Zone for every user should not be needed.
However, if you can't log in I assume you're currently blocked by this. So here's another idea: create it via the shell.
```
$ python manage.py shell
>>> from vespawatch.models import *
```

Create a `FirefightersZone` object:
```
>>> zone = FirefightersZone.objects.create(name='my zone')
```

Find your user
```
>>> you = User.objects.filter(username='peterdesmet').first()
```

Create a profile for your user
```
>>> Profile.objects.create(user=you, zone=my_zone)
```

As mentioned. It's a temp fix to unblock you.

**Comment by niconoe** [comment]
Thanks @bartaelterman,

I think a simple:

    User.objects.filter(username='peterdesmet').first().save()

might be enough!

**Comment by bartaelterman** [comment]
wicked!

**Comment by peterdesmet** [comment]
Didn't really know how to combine your replies, but it worked with (mainly fixing some typos):

```
$ python manage.py shell
>>> from vespawatch.models import *
>>> zone = FirefightersZone.objects.create(name='my zone')
>>> you = User.objects.filter(username='peter_desmet').first()
>>> Profile.objects.create(user=you, zone=zone)
```

**Comment by niconoe** [comment]
Great. Let's keep the issue opened so I remember to implement a proper fix.

**Comment by niconoe** [comment]
Will be solved later, as part of #54.

**Comment by niconoe** [comment]
Improved temporary fix:

    $ python manage.py shell
    >>> from django.contrib.auth.models import User 
    >>> from vespawatch.models import Profile
    >>> Profile.objects.create(user=User.objects.filter(username='niconoe').first())
